_docs/data-model/v1/device-meta.md

```diff
@@ -11,9 +11,96 @@ The "type" of a deviceMeta event is defined by the `subType` field.
+
+- `low_insulin` for low reservoir/cartridge alarms
+- `no_insulin` for empty reservoir/cartridge alarms
+- `low_power` for low battery alarms
+- `no_power` for out of battery alarms
+- `occlusion` for occlusion (blockage of infusion set) alarms
+- `auto_off` for when the pump stops all delivery due to inactivity for a period longer than the user's programmed threshold (if any)
+- `over_limit` for when insulin delivery has surpassed any of a user's programmed maximum bolus, basal, or hourly delivery thresholds
+
+Many if not all `alarm` events will include a `payload` object with more information about the alarm that is device-specific. For example, a `low_insulin` alarm may have a `units_left` field in its `payload` to record the number of units of insulin that were remaining in the insulin pump's reservoir at the time of the alarm.
+
+In addition, a `payload` object is *required* when `alarmType` is `other`; this value is used to capture all alarms that are device-specific. For example, a pod expiration alarm is specific to the Insulet OmniPod insulin delivery system, and a 'pump body disconnected' alarm is specific to an Asante Snap insulin pump. The `payload` object should include all information that could be relevant to anyone wishing to audit the history and performance of the insulin pump in question.
+
+### Storage/Output Format
+
+The storage and output format for this datum is exactly what was initially ingested.  There are no modifications performed.
+
+## Prime
+
+A prime event represents the priming of either an infusion line (tubing) or an infusion cannula. It looks like
+
+~~~json
+{
+  "type": "deviceMeta",
+  "subType": "prime",
+  "primeTarget": tubing_or_cannula,
+  "volume": optional_units_delivered,
+  "time": see_common_fields,
+  "deviceId": see_common_fields,
+  "uploadId": see_common_fields
+}
+~~~
+
+`primeTarget` can have two values depending on the object of the priming action - either `tubing` for an infusion line prime or `cannula` for a cannula prime.
+
+The `volume` field is optional. It should be included if the data specifies the volume of insulin delivered in the course of the priming.
+
+### Storage/Output Format
+
+The storage and output format for this datum is exactly what was initially ingested.  There are no modifications performed.
+
+## Reset
+
+A reset event represents any event in an insulin delivery system that implies a return to a device state not yet ready to deliver insulin. This varies depending on the type of insulin delivery device. For conventional syringe-type insulin pumps, this will be a rewind event. For an Insulet OmniPod system, it is a new pod activation event. For an Asante Snap system, it is a new pump body event. This event often implies a suspension of insulin delivery; in the case that the device data includes a reset event but does not include a separate indication of insulin delivery suspension, a `status` event should also be uploaded to the platform.
```

Comment:
Ah, I was thinking it should only be included iff the status event is inferred. If it's explicit, it may/probably have a slightly different timestamp and for me at least then fails the gut check test of "this is the same event".